src/main/java/de/adesso/kicker/match/controller/MatchController.java

```diff
@@ -67,11 +67,14 @@ public ModelAndView postAddMatch(@Valid Match match, BindingResult bindingResult
     }
 
     private ModelAndView defaultAddMatchView(ModelAndView modelAndView) {
-        modelAndView.addObject("match", new Match());
-        modelAndView.addObject("users", userService.getAllUsers());
-        modelAndView.addObject("currentUser", userService.getLoggedInUser());
-        modelAndView.addObject("notifications",
-                notificationService.getNotificationsByReceiver(userService.getLoggedInUser()));
+        var match = new Match();
```

Comment:
Is this a better way to program?
looks a bit more readable but also less performant

Reply:
I removed one unnecessary database call. This should be more performant than before.

Reply:
i mea that every object gets saved into a new variable before it is passed to the model

Reply:
The compiler will optimise that most likely so it doesn't really matter.